### PR TITLE
enh(CPlusPlus): reverse string

### PR DESCRIPTION
**Pull request by ganesh1828**
String Reversal program is modified without using any built in function and used proper comments.

### CODE REVIEW

**Comment by ming-tsai** [comment]
Hi @ganesh1828,
Please remove the change of `algorithms/CPlusPlus/Recursion/gcd.cpp` and use the pull request template on this pull request

File: algorithms/CPlusPlus/Recursion/gcd.cpp [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        swap(a, b);
+    }
+    return a;
+}
+int main() {
+    int n1 = 56, n2 = 65;
+    cout << "GCD of " << n1 << " and " << n2 << " is " << non_rec_gcd(n1, n2); 
+	return 0;
+}
+
```
Comment (aayushjain7) [inline algorithms/CPlusPlus/Recursion/gcd.cpp:21]: provide examples and time complexity.

File: algorithms/CPlusPlus/Strings/string-reverse.cpp [hunk 2]
```diff
@@ -1,12 +1,30 @@
...
 int main() {
     string s;
     cin>>s;
-reverse(s.begin(),s.end());
-cout<<s;
+	s=reverse_string(s);//calling reverse_string function and passing string as parameter
+	cout<<s;
 return 0;
 
 }
```
Comment (aayushjain7) [inline algorithms/CPlusPlus/Strings/string-reverse.cpp:30]: provide examples and time complexity.

**Review by aayushjain7 — CHANGES_REQUESTED** [review]
update readme.MD file for both the files

**Comment by ming-tsai** [comment]
Hi @ganesh1828,
Any updates?

**Comment by ming-tsai** [comment]
Hi @ganesh1828,
This pull request will be closed when making a change please re-open this pull request.

**Pull request closed without merging**